### PR TITLE
feat: Entity explorer - page container resize, fluid typography

### LINKED ISSUE #11284: [Task] Resizable pages container | Entity explorer.
Typography changes.

### PR DESCRIPTION
**Pull request by albinAppsmith**
## Description

 This PR includes the responsive font size for entity explorer and page container resize changes.

Fixes #11284

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Tested in chrome dev tools for fluid typography changes

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: feat/entity-explorer-resize-page-fluid-typography 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 56.62 **(0)** | 38 **(0)** | 36.17 **(0.02)** | 56.84 **(0)**
 :red_circle: | app/client/src/pages/Editor/Explorer/Entity/index.tsx | 91.67 **(-0.09)** | 85.71 **(0)** | 83.33 **(-0.88)** | 91.25 **(-0.11)**
 :green_circle: | app/client/src/pages/Editor/Explorer/Pages/index.tsx | 84.29 **(1.24)** | 73.08 **(-1.92)** | 47.06 **(4.2)** | 83.82 **(0.77)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**
 :sparkles: :new: | **app/client/src/utils/hooks/useResize.tsx** | **45.95** | **40** | **66.67** | **48.39**</details>

### CODE REVIEW

**Review by jsartisan — DISMISSED** [review]

File: app/client/src/utils/hooks/useResize.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React, { MutableRefObject } from "react";
```
[the remark this answers is not in the document]
Comment (albinAppsmith) [reply app/client/src/utils/hooks/useResize.tsx:1]: we can do that, will do that in a separate PR

**Comment by mohanarpit** [comment]
Closing this PR as the issue has been fixed in #13576

**Pull request closed without merging**